includes/class-http.php

```diff
@@ -60,9 +60,9 @@ public static function post( $url, $body, $user_id ) {
 	 *
 	 * @return array|WP_Error The GET Response or an WP_ERROR
 	 */
-	public static function get( $url, $user_id ) {
+	public static function get( $url ) {
 		$date = \gmdate( 'D, d M Y H:i:s T' );
-		$signature = Signature::generate_signature( $user_id, 'get', $url, $date );
+		$signature = Signature::generate_signature( -1, 'get', $url, $date );
```

Comment:
I would not send every request signed from the blog-author! Is this even accepted by Mastodon and others?

Reply:
It is a nice idea to maybe use it for getting remote data, but not for sending `create` activities I think.

Reply:
This is just for get requests, mastodon does the same, providing a minimum user anonymity

Reply:
I'm using `-1` for the application actor, leaving `0` for the blog-author

Reply:
if the number of non-user rise, should we use constants instead? to not get confused?

Reply:
Good idea!